Dockerfile

```diff
@@ -0,0 +1,39 @@
+# syntax=docker/dockerfile:1
+ARG GCC_VERSION=11.2.0
+ARG CMAKE_VERSION=3.22.0
+ARG GRPC_VERSION=1.43.0
+
+FROM iomz/cmake:$CMAKE_VERSION AS cmake
+FROM iomz/grpc-protoc:$GRPC_VERSION AS grpc
```

Comment:
Since the gRPC image takes insanely long to build, I cheated here. I will come back to this with #54.